docs/provisioner/08-02-provisioning-gardener.md

```diff
@@ -138,10 +138,10 @@ This tutorial shows how to provision clusters with Kyma Runtimes on Google Cloud
               }
             }
             kymaConfig: { # Optional; if you don't provide it, a cluster without Kyma is provisioned
```

Comment:
kymaConfig section should not be provided in the case of provisioning version 2.X and higher